HWMgmt/OCPBaselineHardwareManagement.v1_1_0.json

```diff
@@ -0,0 +1,493 @@
+								"LimitInWatts": {},
+								"LimitException": {}
+							}
+						}
+					}
+				}
+			}
+		},
+		"PowerSubsystem": {
+			"PropertyRequirements": {
+				"CapacityWatts": {
+					"Purpose": "Equivalent for Power.PowerControl.PowerCapacityWatts.",
+					"ReadRequirement": "Recommended"
+				}
+			}
+		},
+		"Sensor": {
+			"Purpose": "Sensor equivalent Power.PowerControl.PowerConsumedWatts.",
+			"PropertyRequirements": {
+				"ID": {
+ 					"Values": ["TotalPower"]
+        		},
+				"Reading": {},
+				"ReadingUnits": {}
+			}
+		},
+		"Control": {
+			"Purpose": "Sensor equivalent Power.PowerControl.PowerLimit.",
+			"PropertyRequirements": {
+				"ID": {
+ 					"Values": ["PowerLimit"]
+        		},
+				"SetPoint": {},
+				"SetPointUnits": {}
+			}
+		},
+		"SessionService": {},
+		"Thermal": {
+			"MinVersion": "1.1.0",
+			"PropertyRequirements": {
+				"Fans": {
+					"ReadRequirement": "IfImplemented",
+					"PropertyRequirements": {
+						"Status": {},
+						"Name": {
+							"ReadRequirement": "Recommended",
+							"Purpose": "Transition from the deprecated 'FanName' to 'Name'."
+						},
+						"Reading": {
+							"ReadRequirement": "Recommended"
+						},
+						"ReadingUnits": {
+							"ReadRequirement": "Recommended"
+						}
+					}
+				},
+				"Temperatures": {
+					"ReadRequirement": "IfImplemented",
+					"PropertyRequirements": {
+						"ReadingCelsius": {},
+						"UpperThresholdFatal": {
+							"ReadRequirement": "Recommended"
+						},
+						"UpperThresholdCritical": {
+							"ReadRequirement": "Recommended"
+						},
+						"UpperThresholdNonCritical": {
+							"ReadRequirement": "Recommended"
+						},
+						"Status": {}
+					}
+				},
+				"Redundancy": {
+					"ReadRequirement": "IfImplemented",
+					"PropertyRequirements": {
+						"MemberId": {},
+						"RedundancySet": {},
+						"Mode": {},
+						"Status": {},
+						"MinNumNeeded": {},
+						"MaxNumSupported": {
+							"ReadRequirement": "Recommended"
+						}
+					}
+				}
+			}
+		},
+
+		"Sensor": {
+			"Purpose": "Sensor for the power consumed.",
+			"PropertyRequirements": {
+				"ID": {
+ 					"Values": ["TotalPower"]
+        		},
+				"ReadingType": {},
+				"Reading": {},
+				"ReadingUnit": {}
+			}
+		},
+
+		"ThermalSubsystem": {
+			"MinVersion": "1.0.0",
+			"PropertyRequirements": {
+				"ThermalMetics": {},
+				"Fans": {},
+				"FanRedundancy": {
+					"ReadRequirement": "IfImplemented",
+					"PropertyRequirements": {
+						"RedundancyType": {},
+						"MinNeededinGroup": {},
+						"MaxSupportedInGroup": {
+							"ReadRequirement": "Recommended"
+						},
+						"RedundancyGroup": {},
+						"Status": {}
+					}
+				}
+			}
+		},
+		"ThermalMetric": {
+			"Purpose": "Equivalent of Thermal.Temperatures",
+			"PropertyRequirements": {
+				"TemperatureSummaryCelsius": {
+ 					"PropertyRequirements": {
+						"Exhaust": {
+							"ReadRequirement": "Recommended"
+						}
+					}
+        		}
+			}
+		},
+		"Fan": {
```

Comment:
Should we add `"ReadRequirement": "IfImplemented"`? There could be designs without fans.

Reply:
Agree. Changed.